File: class09/java/AP0910-EscopoVariaveis.java
import java.util.Scanner;
public class EscopoVariaveis {
    static final double PI = 3.14159;
    static double calcularArea(double raio) {
        double area = PI * raio * raio;
        return area;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Constante PI = " + PI + " (acessível em todo o programa)");
        System.out.print("Digite o raio: ");
        double raio = sc.nextDouble();
        System.out.printf("Área do círculo: %.2f%n", calcularArea(raio));
        sc.close();
    }
}

File: class09/python/AP0910-EscopoVariaveis.py
# EscopoVariaveis — Aula 09
# Demonstra escopo local vs global

PI = 3.14159  # constante por convenção
print(f"Constante PI = {PI} (acessível em todo o programa)")
def calcular_area(raio):
    area = PI * raio ** 2  # area é variável local
    return area
raio = float(input("Digite o raio: "))
resultado = calcular_area(raio)
print(f"Área do círculo: {resultado:.2f}")
